src/huggingface_hub/utils/_cache_manager.py

```diff
@@ -205,6 +287,114 @@ def size_on_disk_str(self) -> str:
         """
         return _format_size(self.size_on_disk)
 
+    def delete_revisions(self, *revisions: str) -> DeleteCacheStrategy:
```

Comment:
The name of the method sounds as if it is deleting the revisions. Maybe it should be renamed to something that better describes it.

Reply:
Do you have a suggestion? I agree with you but on the other hand I have not found a naming that I preferred more. I see the `delete_revisions().execute()` a bit as what an ORM API looks like `db.insert(...); db.flush(...)`.

Reply:
I can't think of a concise, expressive method name unfortunately. Feel free to go ahead with this if you feel it's ok.

Reply:
I also thought if a user do a `delete_revisions()` and doesn't realize the cache has not been deleted it's less of a problem than deleting something without noticing. That's why I didn't want to have an optional "dry_run" argument in the method for example.